client/dashboard/me/billing-purchases/payment-method-selector/index.tsx

```diff
@@ -0,0 +1,230 @@
+import { SubscriptionBillPeriod } from '@automattic/api-core';
+import { userPaymentMethodsQuery, assignPaymentMethodMutation } from '@automattic/api-queries';
+import {
+	CheckoutProvider,
+	CheckoutPaymentMethods,
+	CheckoutFormSubmit,
+} from '@automattic/composite-checkout';
+import styled from '@emotion/styled';
+import { useQuery, useMutation } from '@tanstack/react-query';
+import { Card, CardBody, Icon } from '@wordpress/components';
+import { useDispatch } from '@wordpress/data';
+import { __, sprintf } from '@wordpress/i18n';
+import { info as infoIcon } from '@wordpress/icons';
+import { store as noticesStore } from '@wordpress/notices';
+import clsx from 'clsx';
+import { useCallback } from 'react';
+import { Notice } from '../../../components/notice';
+import { creditCardHasAlreadyExpired, isAkismetProduct } from '../../../utils/purchase';
+import { assignExistingCardProcessor } from '../payment-methods';
+import getPaymentMethodIdFromPayment from './get-payment-method-id-from-payment';
+import TosText from './tos-text';
+import type { Purchase } from '@automattic/api-core';
+import type { PaymentMethod } from '@automattic/composite-checkout';
+
+const TOSItemWrapper = styled.div`
```

Comment:
Can we remove the styled component in this PR? We prefer default `@wordpress/components` with minimal inline styles and, if necessary, styles in a .scss file.